### ISSUE TITLE
Unable to find plugin 'SERIALCOM' (referenced via ProjectName.uproject). Install it and try again, or remove it from the required plugin list.

### ISSUE THREAD

**Issue by anurajbh** [videofeedback/Unreal_Engine_SerialCOM_Plugin#15]
Hello,

I am running into an issue where after importing the plugin, if I try to make a C++ class or compile, I get the error that the plugin is not available.

I have checked that the plugin is installed. Before I made the C++ class, I was able to make the Arduino I have work with Blueprints. The moment I made a blank Actor class in C++, I started getting that error.

What's more, now I find myself unable to even open the project.

**Comment by videofeedback** [comment]
Did you installed Visual Studio 2022? ... You need the latest version to compile the project.
I suggest to uninstall Visual Studio first, and then re-install Visual Studio 2022 with the Unreal Engine option.
https://visualstudio.microsoft.com/vs/community/

**Comment by anurajbh** [comment]
> Did you installed Visual Studio 2022? ... You need the latest version to compile the project. I suggest to uninstall Visual Studio first, and then re-install Visual Studio 2022 with the Unreal Engine option. https://visualstudio.microsoft.com/vs/community/

Yes, I have VS 2022 and I did install it with the Unreal Engine option when I got it.

Would you suggest updating it and then trying?

**Comment by Gerardo-Dev-unity** [comment]
Hello, I am experiencing the same error when packaging the project for Windows. It gives me the error referred to by anurajbh. I updated Visual Studio, but I still haven't been able to package it. I am using Unreal Engine version 5.4.3, and I am not sure if this is the problem. If so, which version should I install to be able to package it? Thank you very much! By the way, excellent work with this plugin, it is amazing!
